ddl/index_merge_tmp.go

```diff
@@ -180,7 +180,7 @@ func (*mergeIndexWorker) String() string {
 	return typeAddIndexMergeTmpWorker.String()
 }
 
-func (*mergeIndexWorker) GetTask() (*BackfillJob, error) {
```

Comment:
Still not support merge back task run in dist mode？

Reply:
Yes, it will be in the next PR.